src/drivers/mct/shr/seq_timemgr_mod.F90

```diff
@@ -1000,6 +1017,22 @@ subroutine seq_timemgr_clockInit(SyncClock, nmlfile, restart, restart_file, pioi
           write(logunit,*) subname,' ERROR: offset not multiple',n,dtime(seq_timemgr_nclock_drv),offset(n)
           call shr_sys_abort()
        endif
+
+       call ESMF_TimeIntervalSet( TimeStep, s=dtime(n), rc=rc )
+       if(CurrTime + TimeStep > minStopTime ) then
+          call shr_sys_abort(subname//"ERROR: Stop time too short, increase STOP_N")
```

Comment:
Please remove this check on stop time. For testing purposes, I can see value in running the model for just a few time steps - and as long as you're not trying to write restart intervals I can't see any reason why we should stop people from doing this.

(Then you can also remove the minStopTime variable.)